blog/_posts/2017-03-14-scala-native-0.1-is-here.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog
+post-type: blog
+by: Denys Shabalin
+title: "Scala Native v0.1 is here!"
+---
+
+We're excited to announce the first release of
+[Scala Native](http://www.scala-native.org), an optimising ahead-of-time
+compiler for Scala built on top of the [LLVM](http://llvm.org) compiler
+infrastracture.
+
+The project was originally announced during Scala Days New York 2016 and
+now has reached a point of relative feature completeness.
```

Comment:
You're going to kill me, but could you maybe add a sentence or two saying something more about Scala Native? What can it do? Why is it useful? What is feature completeness? ...For someone who has no idea about Scala Native's existence, or who doesn't know much about what LLVM is, it would be cool to see why this project matters stated here too. If I don't know what LLVM is but I like Scala, why should I care about this project?